src/wwwroot/controllers/BillingCtrl.js

```diff
@@ -0,0 +1,95 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('dopplerRelay')
+    .controller('BillingCtrl', BillingCtrl);
+
+  BillingCtrl.$inject = [
+    '$scope',
+    '$location',
+    '$rootScope',
+    'auth',
+    '$translate',
+    '$timeout',
+    'settings',
+    'utils'
+  ];
+
+  function BillingCtrl($scope, $location, $rootScope, auth, $translate, $timeout, settings, utils) {
+    var vm = this;
+    $rootScope.setSubmenues([
+      { text: 'submenu_my_profile', url: 'settings/my-profile', active: false },
+    ]);
+    vm.checkExpDate = checkExpDate;
+    vm.submitBilling = submitBilling;
+
+    vm.cc = {number:'', type:{}, mask:''};
+    vm.maskOptions = {
+  	  allowInvalidValue:true //allows us to watch the value
+    };
+    $scope.$watch('vm.cc.number', function(newNumber){
```

Comment:
Maybe it is not necessary to destroy it explicitly, because it is _$scope_ and not _$rootScope_, please verify it yourself and do not add the destroy if it is not necessary.